framework/doc/content/source/postprocessors/ContactDofSet.md

```diff
@@ -0,0 +1,22 @@
+# ContactDofSet
```

Comment:
This should probably be named `ContactDOFSetSize` or  `ContactDOFNumber` to indicate that it outputs a number rather than a set.